### ISSUE TITLE
Feature request: Convert hashtags used in dataset + history names into real hashtags

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#7106]
E.g. that 

![grafik](https://user-images.githubusercontent.com/458683/49930177-f22a5480-febb-11e8-8f38-6e499a73824b.png)

Could become

![grafik](https://user-images.githubusercontent.com/458683/49930219-0b330580-febc-11e8-9bfa-2f197f2b9eee.png)

automatically. (Also maybe show history tags or hashtags using the new colours.)

**Comment by mvdbeek** [comment]
Seems a bit risky, what if `#` should be part of the history / dataset name ?

**Comment by hexylena** [comment]
Part of history name ­- unlikely

Part of dataset name, I was thinking "only if the dataset is renamed manually it might be a nice convenience." (But this sort of assumes that dataset renaming is easier than a 3-4 click operation.)

So maybe not useful? But thought I would put it out there.

**Comment by mvdbeek** [comment]
I don't know, who knows how people are organising their histories (or workflow PJAs), I'd rather not see it. I think I'd be super annoyed if my previous naming scheme used `#`.

**Comment by mvdbeek** [comment]
I think we can leave this open and see if people do actually think it'd be nice / have different ideas ? When editing interactively we could have some fancy things that suggests putting this into tags ?
I didn't mean to sound too negative. I can see how that'd reduce unnecessary clicking.

**Comment by hexylena** [comment]
Yeah, I was mostly thinking of my task manager which auto-completes certain things into more useful data, e.g. "!High" gets converted into a high priority, or "#blah" tags the task as that. Maybe someone else will find it interesting, maybe not
